test/provisioning/account_spec.js

```diff
@@ -25,13 +26,13 @@ describe('account API - Provisioning', function () {
     }
 
     // Create a sub account(sub cloud)
-    let res = await cloudinary.provisioning.account.create_sub_account('jutaname' + Date.now(), 'jutaname' + Date.now(), {}, true).catch((err) => {
+    let res = await cloudinary.provisioning.account.create_sub_account(CLOUD_NAME_PREFIX + Date.now(), CLOUD_NAME_PREFIX + Date.now(), {}, true).catch((err) => {
```

Comment:
@patrick-tolosa , I guess the name has no meaning in this case, but It's better that (if anyone will look at our tests as example of usage) we will set the name and the cloudName to be similar.

Reply:
What do you mean by that? 

Similar to each other? or should the cloud name be called "cloudName"?

Reply:
Since you're calling Date.now() twice, you might get different values, hence your cloudName and name will have different values. 
Not very interesting in our case, but if someone will copy paste from here...
Just extract to a variable